src/js/tracks/text-track-settings.js

```diff
@@ -489,15 +486,11 @@ class TextTrackSettings extends ModalDialog {
   }
 
   content() {
-    const settings = createEl('div', {
-      className: 'vjs-tracksettings'
-    }, undefined, [
+    return [
```

Comment:
Having this wrapper `div` made sense when this wasn't a `ModalDialog` but since it is now and we're moving toward a breaking change anyway, might as well clean up the DOM.